README.md

```diff
@@ -297,7 +299,36 @@ either enable "debug" dracut module or add `dracut_install netstat ip` line to
 (for "rdsosreport.sh") show - there can be no default route, whatever interface
 naming mixup, no traffic (e.g. unrelated connection issue), etc.
 
-## 5.4 Report a bug
+## 5.4 *dracut: dropbearconvert for ED25519 key failed*
+
+In the case that your dropbearconvert does not support ED25519 encryption (e.g CentOS Stream), you will need to install
+dropbear from source upstream of [this PR](https://github.com/mkj/dropbear/pull/91).  We will use the `DROPBEAR_2020.81` tag in the example below.
+
+1. Clone [dropbear git repo](https://github.com/mkj/dropbear)
+
+2. Check out the latest release tag
+   ```
+   git checkout DROPBEAR_2020.81
+   ```
+
+2. Install from source. 
+   ```
+   ./configure 
+   make
+   sudo make install
+   ```
+
+3. Now re-run `dracut --force`
+
+Since the sudo user only has the following items in path `/sbin:/bin:/usr/sbin:/usr/bin` and our version of dropbear is installed in `/usr/local/bin`, we need to extend the path of the sudo user for this command to ensure the right version of dropbear is used. 
+
+We can do this with the following command:
+```
+sudo PATH="/usr/local/bin:$(sudo bash -c 'echo "$PATH"')' dracut --force
+```
+   
+
```

Comment:
None of this is within the scope of the troubleshooting for this dracut module. At most, this should consist of something like
> If your `dropbear` installation is older than version XXXX, it will not support ED25519 keys and installation of this module will fail with the message `dracut: dropbearconvert for ED25519 key failed`. Make sure you install a sufficiently modern version of `dropbear` and re-run `dracut --force` to regenerate your initramfs. (Note: if you install a local copy of `dropbear` and do not remove your outdated system package, make sure that the new version of `dropbearconvert` appears first in your `$PATH`.

I intend to make the `keyTypes` variable at https://github.com/dracut-crypt-ssh/dracut-crypt-ssh/blob/master/modules/60crypt-ssh/module-setup.sh#L21 a configuration option (*e.g.*, `dropbear_keytypes`) to allow control over which keys are installed in the initramfs. When that capability is added, we can amend the troubleshooting text to note the option of simply disabling ED25519 keys.

Reply:
#52